lib/rules/web/admin_console/4.3/logs.xyaml

```diff
@@ -0,0 +1,11 @@
+# logs page
+switch_to_second_container:
+  elements:
+  - selector:
+      xpath: //button[contains(@class,'pf-c-dropdown__toggle')]//span[contains(text(),'<current_container_name>')]
```

Comment:
Suggest to use switch_to_other_container as rule name.
 And no need to pass <current_container_name> in xpath, try to use:
xpath: //button[@data-test-id='dropdown-button']